demo/molecule/accordion/playground

```diff
@@ -0,0 +1,63 @@
+const icon = GetIcon()
+const showText = 'Show'
+const hideText = 'Hide'
+const height = 70
+const maxHeight = 200
+
+function Text() {
+    return(
+      <div label='Title'>
+        <p style={{margin:0}}>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam dictum magna non diam euismod blandit et eu ex. Vivamus pulvinar sodales tincidunt. Proin venenatis tristique quam, quis vehicula eros volutpat sed. Etiam sed tristique ante. Aenean commodo erat quis pulvinar luctus. Pellentesque ultricies lorem vitae ante euismod, at imperdiet est euismod. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. In dignissim porttitor sem, a varius nisl ullamcorper ut. Vivamus lacinia, quam eu placerat tempus, velit massa vulputate turpis, sit amet bibendum risus massa sit amet urna. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Praesent finibus lobortis blandit. Vivamus scelerisque blandit purus a suscipit. Nunc mi elit, condimentum eget pulvinar eu, lacinia vitae ligula. Sed sit amet eros auctor ipsum tincidunt hendrerit ac mollis justo. Ut ac sagittis ipsum.</p>
+      </div>
+  )
+}
+
+function GetIcon () { 
+  return  (
+      <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="16px" height="16px" viewBox="0 0 32 32">
+          <path d="M30,9l1,1c1,1,1,2,0,2L17,25c-1,1-2,1-2,0 L2,12c-1-1-1-2,0-2l1-1c1-1,2-1,2,0L16,20L28,9 C28,8,29,8,30,9z"></path>
+      </svg>
+  )
+}
+
+function Demo ({children}) {
+return <div style={{width: '100%', padding: 20}}>{children}</div>
+}
+
+function DemoWrapper ({children}) {
+return <div style={{display: 'flex', flexWrap: 'wrap'}}>{children}</div>
+}
+
+return (
+  <div>
+    <h1>Accordion</h1>
+    <DemoWrapper >
+      <Demo>
+        <h2>Accordion with transition and autoclose</h2>
+        <div style={{backgroundColor: '#fff', fontSize: 14, padding: 16, textAlign: 'left'}}>
+          <MoleculeAccordion maxHeight={100} withAutoClose={true} withTransition={true} icon={GetIcon()}>
+            <div label='Title'>
```

Comment:
There is already a Tab subcomponent for accordion but it is not exported as a component. 
If I export it then it would show up in the SUI-components list and would be another dependency, am I wrong?

Reply:
No, if it doesn't have its own folder with its package.json but is inside your Accordion component, it won't show up in the list